crates/neon/src/types_impl/mod.rs

```diff
@@ -1154,6 +1154,23 @@ impl JsFunction {
     }
 }
 
+pub(crate) unsafe fn call_local<'a, 'b, C: Context<'a>, T, AS>(
+    cx: &mut C,
+    callee: raw::Local,
+    this: Handle<'b, T>,
+    args: AS
+) -> JsResult<'a, JsValue>
+where
+    T: Value,
+    AS: AsRef<[Handle<'b, JsValue>]>,
+{
+    let (argc, argv) = unsafe { prepare_call(cx, args.as_ref()) }?;
+    let env = cx.env().to_raw();
+    build(cx.env(), |out| unsafe {
```

Comment:
```rust
match status {
    sys::Status::FunctionExpected => todo!("Write code to convert to an exception"),
    sys::Status::PendingException => return Err(Throw::new()),
    status => assert_eq!(status, sys::Status::Ok),
}
```

https://github.com/neon-bindings/neon/blob/5d16c4d0c040c7d81565673ba4b42929922e03e1/crates/neon/src/sys/bindings/types.rs#L109

Reply:
Turns out it's `InvalidArg`, not `FunctionExpected`, but otherwise this was perfect guidance.